content/en/tracing/service_catalog/integrations.md

```diff
@@ -29,7 +29,9 @@ You can connect any service in your [PagerDuty Service Directory][1]. You can ma
 
 2. Get your PagerDuty API access key as described in their [API Access Key][3] documentation.
 
-3. Follow the [integration configuration instructions][4] to finish configuring it.
+3. Enter the API Access Key on [Pagerduty Integration Setup][4] to finish configuring it.
+![image](https://github.com/DataDog/documentation/assets/37256626/eacc9417-c3bc-4091-b87e-e03e1115b20b)
```

Comment:
@hyperloglogy Can you add the PNG image file to the `static/images/tracing/service_catalog` folder and use an image reference. If this is a common question customers are confused about, add the image. If they can easily find this with the link, leave the image out.
```
{{< img src="path/to/your/image-name-here.png" alt="Your image description" style="width:100%;" >}}
```

Reply:
we decided to add an image because customers are frequently confused about the step #3. I'll add the image to the dir. thanks.